### PR TITLE
docs(apim): update apim readme file

### PR DESCRIPTION
**Pull request by XiaofuHuang**
Update APIM readme file for Arm Support

### CODE REVIEW

File: packages/fx-core/templates/plugins/resource/apim/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Publish Teams Backend (Azure Functions) to Azure API Management
 
-Azure API Management (APIM) is used to create consistent and modern API gateways for existing backend services. With Teams Toolkit or TeamsFx CLI, you can easily publish your backend APIs (Azure Functions) to existing or new APIM instance.
+Azure API Management (APIM) is used to create consistent and modern API gateways for existing backend services. With Teams Toolkit or TeamsFx CLI, you can easily publish your backend APIs (Azure Functions) to APIM instance.
 
 ## Prerequisite
```
Comment (dilin-MS) [inline packages/fx-core/templates/plugins/resource/apim/README.md:6]: line#8 
```An M365 account, if you do not have an M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)```
should be `apply for one from xxx`

Comment (dilin-MS) [reply packages/fx-core/templates/plugins/resource/apim/README.md:6]: line10 
```
Ensure the resource provider 'Microsoft.ApiManagement' is registered for the subscription by following [this document]
```
should be `by following [document](xxx)`. Please help remove the `this`.

**Review by dilin-MS — DISMISSED** [review]

File: packages/fx-core/templates/plugins/resource/apim/README.md (hunk 1 above)
Comment (XiaofuHuang) [reply packages/fx-core/templates/plugins/resource/apim/README.md:6]: Thanks!

File: packages/fx-core/templates/plugins/resource/apim/README.md [hunk 2]
```diff
@@ -34,7 +30,7 @@ Simply deploy your project to the cloud when it’s ready by following these ste
 - Deploy to the cloud
 
 You can do this using the Teams Toolkit in Visual Studio Code or using the TeamsFx CLI:
-| Using Teams Toolkit| Using TeamsFx CLI|
+| From Visual Studio Code | From TeamsFx CLI|
```
Comment (dilin-MS) [inline packages/fx-core/templates/plugins/resource/apim/README.md:33]: If you change `Using` to `From`, do we need to update the same wording in above section?

Comment (XiaofuHuang) [reply packages/fx-core/templates/plugins/resource/apim/README.md]: Change it same to the tab app readme

**Review by dooriya — APPROVED** [review]

**Pull request merged**